### ISSUE TITLE
A suggestion to write injected code in one file

### ISSUE THREAD

**Issue by homekuanyi** [johnno1962/InjectionIII#30]
Hi, johnno1962, thanks for your great work. 

I would like to use InjectionIII in a team project, so I should not write the injection code in AppDelegate.swift. ( other team member might not use InjectionIII )
```
Bundle(path: "/Applications/InjectionIII.app/Contents/Resources/iOSInjection.bundle")?.load()
```

So my suggestion is that we should write all injection code in a new file named Injection.swift, and it's content looks like:
```
import UIKit

#if DEBUG //1

extension UIViewController { //5

    @objc func injected() { //2
        for subview in self.view.subviews { //3
            subview.removeFromSuperview()
        }
        viewDidLoad() //4
    }
}

extension AppDelegate {
    
    private static let installInjectionOn: Void = {
        Bundle(path: "/Applications/InjectionIII.app/Contents/Resources/iOSInjection.bundle")?.load()
    }()
    
    override open var next: UIResponder? {
        AppDelegate.installInjectionOn
        return super.next
    }
}

#endif
```

And for other people who want to use InjectionIII, it may be easier to add a single file to his project.

**Comment by johnno1962** [comment]
Good idea, perhaps you could make this a CocoaPod?

**Comment by zenangst** [comment]
@homekuanyi hey mate, seems like you have/had the same predicament as me. You want to use injection because it's awesome but you don't want to clutter up the code base of your colleagues who haven't seen the value of injection just yet. This was one of the motivating factors behind a framework I created for easily integrating injection into any application. You can find the project here - https://github.com/zenangst/Vaccine. It might help your situation, it might not... Just thought I'd bring it up as it seems like you are trying to solve a similar issue.

**Comment by zenangst** [comment]
The first version of Vaccine is now implemented in version 1.4, you can find the implementation here - https://github.com/johnno1962/InjectionIII/blob/master/InjectionBundle/Vaccine.swift

Closing this issue now :)
